data/recipes.yaml

```diff
@@ -173,6 +198,17 @@
   time: 9
   weight: 160
 
+- in:
+  - [1, mountain]
+  out:
+  - [16, lodestone]
+  - [8, iron ore]
+  - [1, mountain tunnel]
+  required:
+  - [1, metal drill]
+  time: 9
+  weight: 10
+
```

Comment:
Note that this means lodestones are a non-renewable resource.  I think that is perfectly OK (there should be plenty of them available), but it's worth noting.

Reply:
Based on what I read on [Wikipedia](https://en.wikipedia.org/wiki/Compass#Magnetic_needle), I planned to make the compass recipe take lodestone as a catalyst.

The idea is that you rub the iron needle with a lodestone and it becomes magnetized.

So you really only need one. :slightly_smiling_face:

Reply:
Ah, OK, makes sense!